src/engraving/accessibility/accessibleitem.cpp

```diff
@@ -113,7 +108,7 @@ const IAccessible* AccessibleItem::accessibleParent() const
         return nullptr;
     }
 
-    return static_cast<EngravingItem*>(p)->accessible();
+    return static_cast<EngravingItem*>(p)->accessible().get();
```

Comment:
any reason not change signature to return smart pointer?

Reply:
because we don't store this object in this class, we get it from element and just return

Reply:
Sure, but now you've lost track of the fact that there's a copy of the pointer somewhere...

Reply:
Thanks, I created tech debt. We can't implement it now because it will require a lot of changes
https://github.com/musescore/MuseScore/issues/12208